tests/components/watchyourlan/test_sensor.py

```diff
@@ -0,0 +1,62 @@
+"""Test the WatchYourLAN sensor platform."""
+
+from unittest.mock import AsyncMock, patch
+
+from homeassistant.core import HomeAssistant
+
+
+async def test_setup_sensors(hass: HomeAssistant, mock_config_entry) -> None:
```

Comment:
Consider using snapshot tests. If you use `snapshot_platform`, you can execute `pytest ./tests/components/watchyourlan --snapshot-update` and it will generate a `.ambr` file for you with all the data structures fixated